cmd/bundle/init.go

```diff
@@ -109,6 +110,24 @@ func getUrlForNativeTemplate(name string) string {
 	return ""
 }
 
+func getFsForNativeTemplate(name string) (fs.FS, error) {
```

Comment:
```suggestion
func getFsForBuiltInTemplate(name string) (fs.FS, error) {
```

This can be confusing because we also have the `nativeTemplates` slice above, which are really first-party templates. To keep this explicit, we should rename `nativeTemplates` -> `firstPartyTemplates` as well (along with the associated methods).

Reply:
Good point. This will require a bit more work than just the rename, though, so I will address this in a follow-up PR.